src/test/ui/nll/borrowed-temporary-error.stderr

```diff
@@ -9,7 +9,13 @@ LL |     });
 LL |     println!("{:?}", x);
    |                      - borrow later used here
    |
-   = note: consider using a `let` binding to create a longer lived value
+help: consider using a `let` binding to create a longer lived value
+   |
+LL ~     let binding = (v,);
+LL ~     let x = gimme({
+LL |         let v = 22;
+LL ~         &binding
```

Comment:
This test shows the limitations of this approach. Adding a let binding will never actually work here, we need to move the `v` declaration (and any operations on it) out of the block as well. I think that we can easily check for these cases and only emit the `note` instead...